D1673/P1673.bs

```diff
@@ -1401,9 +1401,10 @@ The types of `x(i)` and `y(i)` might be more complicated;
 e.g., `x(i)` might be a polynomial with `double` coefficients,
 and `y(i)` a polynomial with `float` coefficients.
 If those polynomials use expression templates,
-then the expression `x(i) + x(i)` might have a completely different type
-than `decltype(x(i))` (possibly with references removed),
-and might also have a completely different type than `alpha * x(i) + y(i)`.
+then slightly different sum expressions involving `x(i)` and/or `y(i)`
+(e.g., `alpha * x(i) + y(i)`, `x(i) + y(i)`, or `y(i) + x(i)`)
+might all have different types,
+all of which differ from value type of `x` or `y`.
```

Comment:
It's probably worth emphasizing that all these types still have to be convertible (or implicitly convertible, or assignable?) to the output type(s).

Reply:
@opensdh Thanks for the feedback!  That's a good point.

Reply:
I think we need both `assignable_from` and `convertible_to` here.  I don't want implementations to have to create a new `output_value_type` instance just to be able to assign it to the output `mdspan`.

Reply:
Fixed; thanks!